### PR TITLE
Building lag

### PR DESCRIPTION
**Pull request by TC-DarkSlayer**
- Reducing lag by requiring certain distance for you to access the factory buttons/inventories
- reducing health of some factory buildings because they take too long to break
- Less Laggy Forges
- Regular forge smelt things in bulk + code cleanup
- Fixed DrillRigs no extractor bug

### CODE REVIEW

File: TFlippy_TerritoryControl_Structures_Dev/Entities/Structures/CoalMine/CoalMine.as [hunk 1]
```diff
@@ -117,18 +122,21 @@ void onTick(CBlob@ this)
...
-		u32 amount = Maths::Max(1, Maths::Floor(XORRandom(resourceYields[index]) * mod));
-		//print(mod +  " " +amount);
-		
-		if (storage !is null)
-		{
-			MakeMat(storage, this.getPosition(), resources[index], amount);
-		}
-		else if (!this.getInventory().isFull())
+
+		for (u8 i = 0;i<8;i++)
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Structures_Dev/Entities/Structures/CoalMine/CoalMine.as:126]: for (u8 i = 0; i<8; i++)
use spaces, much more readable

**Review by laikadoggirl — COMMENTED** [review]
Why change extractor? Nerfed all of their health for a slightly faster tick rate

**Review by laikadoggirl — COMMENTED** [review]
Forge change is also stupid, requiring double the entry requirements to smelt

**Review by laikadoggirl — COMMENTED** [review]
Tried to sneak in a massive drill rig nerf as well, removing mithril, and massively nerfing range, only tradeoff is making is 4x faster

**Pull request closed without merging**